packages/common/src/TimeUtils.ts

```diff
@@ -7,7 +7,7 @@ dotenv.config();
  * @notice Return average block-time for a period.
  */
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-export async function averageBlockTimeSeconds(lookbackSeconds?: number, networkId?: number): Promise<number> {
+export async function averageBlockTimeSeconds(networkId?: number): Promise<number> {
```

Comment:
Good call. While we're at it, should we change this variable name to chainId rather than network id?

Reply:
Sure but is that more accurate..? I've always been confused about chain ID vs network ID

Reply:
They're almost never different. As far as I understand, chainId protects against replay attacks, whereas network id is how nodes find each other. For applications that operate _outside_ of the consensus layer, I think chain id is the preferred identifier.